src/iwant_bot/storage.py

```diff
@@ -72,3 +72,50 @@ def remove_activity_request(self, request_id, person_id):
             )
             raise exception
         self._requests["activity"].remove(request)
+
+
+class TaskStorage(abc.ABC):
+    @abc.abstractmethod
+    def __init__(self):
+        pass
+
+    @abc.abstractmethod
+    def store_task(self, task_id, task_content):
+        """
+        Stores a task so it can be retreived.
+        """
+        pass
+
+    @abc.abstractmethod
+    def retreive_task(self, task_id):
+        """
+        """
+        pass
+
+    @abc.abstractmethod
+    def retreive_any_task(self):
+        """
+        """
+        pass
+
+
+class MemoryTaskStorage(TaskStorage):
+    def __init__(self):
+        self._tasks = dict()
```

Comment:
`pop` and `popitem` methods throw key errors even with defaultdicts. And since there is no `getitem` method used, defaultdict doesn't make a positive difference.